src/currencymanager.cpp

```diff
@@ -237,6 +237,9 @@ CurrencyDialog::CurrencyDialog(CurrencyManager& manager) : currency_manager_(man
     UpdateTotalWisdomValue();
     layout_->addWidget(new QLabel("Total Scrolls of Wisdom"), curr_row + 1, 0);
     layout_->addWidget(total_wisdom_value_, curr_row + 1, 1);
+    button_close_ = new QPushButton("Close");
```

Comment:
I think it would be better to override `windowFlags`. What unexpected behavior could happen?